.nvmrc

```diff
@@ -1 +1 @@
-4.4.1
+14
```

Comment:
we are at https://github.com/themaven-net/tempest-schema/blob/master/.nvmrc 14.19.3 in tempest-schema

Reply:
The intent here was to match the major version of what's referenced there at least, but I did not want to use a specific version.